rpm_requirements.txt

```diff
@@ -7,4 +6,3 @@ nosync
 python-lxml
 PyYAML
 subversion
-wget
```

Comment:
Don't we need those because of backwards compatibility? We should at least mention them in the building old versions documentation.